lib/mailman/receiver/imap.rb

```diff
@@ -44,17 +44,27 @@ def initialize(options)
 
       # Connects to the IMAP server.
       def connect
-        tries ||= 5
-        if @connection.nil? or @connection.disconnected?
-          @connection = Net::IMAP.new(@server, port: @port, ssl: @ssl)
-          if @starttls
-            @connection.starttls
+        begin
+          if @connection.nil? or @connection.disconnected?
+            @connection = Net::IMAP.new(@server, port: @port, ssl: @ssl)
+            if @starttls
+              @connection.starttls
+            end
+            @connection.login(@username, @password)
           end
-          @connection.login(@username, @password)
+        rescue Net::IMAP::ByeResponseError, Net::IMAP::NoResponseError => e
```

Comment:
Hi, I remove the IMAP logging errros, because in Mailman::Application.run block, we can see the exception message.